test/object-store/sync/session/progress_notifications.cpp

```diff
@@ -588,7 +587,6 @@ TEST_CASE("progress notification", "[sync]") {
             REQUIRE(callback_was_called_2);
 
             // Second callback, last one for first notifier
-            original_downloadable_2 = current_downloaded + current_downloadable;
```

Comment:
This has all changed on the base branch, so there shouldn't be any need to change this file once you resolve the conflicts.